roles/dev/meta/main.yml

```diff
@@ -15,6 +15,7 @@ dependencies:
 - { role: ansible, tags: 'prebake-for-dev' }
 - { role: docker }
 - { role: etcd }
+- { role: consul }
```

Comment:
This might need a bit more work. However, we need consul in the Vagrant boxes so that the netplugin tests can use it.